### PR TITLE
Recent changes with different viewing method plus timetable

### PR DESCRIPTION
**Pull request by Lukasz-Bdn**
1. Different way of viewing school activities for all account types.
2. Consolidated views into few single pages for each role (may require cleaning the /views directory - not done yet).
3. Fixed the account types access: now school admin, teacher and student can only access their school.
4. Implemented simple timetable of lessons for teacher and student.
5. Limited timetable to only show days and hours when lessons are present.

### CODE REVIEW

File: src/main/java/pl/schoolmanager/bean/SchoolAdminAccess.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import pl.schoolmanager.entity.UserRole;
+import pl.schoolmanager.repository.UserRoleRepository;
+
+@Component
+public class SchoolAdminAccess {
+	@Autowired
+	UserRoleRepository userRoleRepo;
+	
+	public boolean checkAccess(Authentication auth, long schoolAdminId) {
```
Comment (przemyslawkonik) [inline src/main/java/pl/schoolmanager/bean/SchoolAdminAccess.java:17]: nice oportunity to make abstract class which SchoolAdminAccess, StudentAccess, TeacherAccess will inherit

File: src/main/java/pl/schoolmanager/controller/StudentViewController.java [hunk 2]
```diff
@@ -54,128 +77,128 @@ public String marks(Model m) {
...
-			return "ni ma";
-		}
-	}
-
+//	@ModelAttribute("mondaySubjects")
+//	public List<String> getMondaySubject() {
+//		HttpSession s = SessionManager.session();
+//		Student thisStudent = (Student) s.getAttribute("thisStudent");
+//		Schedule schedule = this.scheduleRepository.findOneByDivisionAndDay(thisStudent.getDivision(), DayOfWeek.MONDAY);
+//		if (schedule == null) {
```
Comment (przemyslawkonik) [inline src/main/java/pl/schoolmanager/controller/StudentViewController.java:85]: i would avoid to put commented code into repository

**Review by przemyslawkonik — APPROVED** [review]
a lot of effort, good job :+1:

**Review by mateuszlichon — APPROVED** [review]
Well done!

**Pull request merged**